guides/contributor/donations.md

```diff
@@ -0,0 +1,60 @@
+# Donations
+
+Donations of preexisting code fall into two broad categories:
+
+* **Small donations:** Some donations only amount to a single PR and should
+  usually just be contributed as such
+* **Large or complex donations:** Other donations are much larger, require
+  ongoing maintenance of their own, and/or introduce nuanced licensing issues
+
+Large donations – or small donations that turn up complex issues during PR
+review – should be referred to the Technical Committee (TC) by filing an issue
+in this `community` repository and tagging
+`@open-telemetry/technical-committee`. The TC will respond to donation
+proposals **within two weeks** (that is, after having time to meet and discuss
+live). If the TC has not responded to the donation request within that
+interval, the donating party can and should point to this document and request
+guidance at the TC's earliest convenience.
+
+All donated code requires a license compatible with the Apache Software License
+2.0, and donated code will require a change of copyright to reflect the
+OpenTelemetry Authors. The Governance Committee will also ask to review any
+trademarks (like the names of components) the donation can carry and make a
+decision to either remove those trademarks or transfer them to the CNCF.
+
+## Donation process
+
+Broadly, these are the steps the OpenTelemetry Governance and Technical
+Committees follow to handle a prospective donation.
+
+1. Per the above, the donating organization creates a GitHub issue using
+   the "Donation Proposal" form in the `community` repository.
+2. The Governance Committee (GC) will evaluate the proposal to ensure that
+   the donation is aligned with the overall OpenTelemetry project vision
+   and roadmap and has a balanced set of interested contributors and maintainers.
+   The GC is also responsible for driving awareness in the community about
+   the contribution and making sure all interested parties have a chance to
+   object and/or contribute. The GC should work with any appropriate Special Interest
+   Groups or Working Groups to evaluate the donation proposal, consider alternatives,
+   and ensure OTel has the resources required to support the donation. When
+   considering alternatives, the GC should consider at least the CNCF ecosystem,
+   and may also consider other well-known open source projects or alternatives proposed
+   by the community.
+3. If a donation proposal passes the initial GC screening, the Technical Committee (TC)
```

Comment:
```suggestion
3. If a donation proposal passes the initial GC screening, the TC
```
for consistency we can use TC throughout if we want since it was defined above